client/src/main/java/tds/config/ClientTestProperty.java

```diff
@@ -0,0 +1,388 @@
+package tds.config;
+
+import java.sql.Timestamp;
+
+/**
+ * Represents a record from the {#code configs.client_testproperties} table.
+ */
+public class ClientTestProperty {
+    private String clientName;
+    private String assessmentId;
+    private Integer maxOpportunities;
+    private Integer handScoredProject;
+    private Integer prefetch;
+    private Timestamp dateChanged;
+    private Boolean isPrintable;
+    private Boolean isSelectable;
+    private String label;
+    private String printItemTypes;
+    private Boolean scoreByTds;
+    private Boolean batchModeReport;
+    private String subjectName;
+    private String origin;
+    private String source;
+    private Boolean maskItemsBySubject;
+    private Boolean initialAbilityBySubject;
+    private Timestamp startDate;
+    private Timestamp endDate;
+    private Timestamp ftStartDate;
+    private Timestamp ftEndDate;
+    private String accommodationFamily;
+    private Integer sortOrder;
+    private String rtsFormField;
```

Comment:
We don't need to do this now, but I think we should see if we can discern what these fields actually mean and use the full words instead of acronyms.  We can then use aliases in our SQL to make the mapper work as expected.